contrib/dutlink/examples/rpi4_video.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+import logging
+import sys
+import time
+import os
+import os.path
+import click
+
+from jumpstarter.client.adapters import PexpectAdapter
+from jumpstarter.common.utils import env
+
+# initialize client from exporter config
+from jumpstarter.common import MetadataFilter
+from jumpstarter.config.client import ClientConfigV1Alpha1
+
+def on_rpi4(client):
+    dutlink = client.dutlink
+    video = client.video
+    camera = client.camera
+    with PexpectAdapter(client=dutlink.console) as console:
+        # stream console output to stdout
+        console.logfile = sys.stdout.buffer
+        # ensure DUT is powered off
+        dutlink.power.off()
+        with open("camera_off.jpeg", "wb") as f:
+            f.write(camera.snapshot())			
+        click.secho("Writing system image", fg="red")
+
+#        dutlink.storage.write_local_file("2024-07-04-raspios-bookworm-arm64-lite.img")
```

Comment:
```suggestion
        dutlink.storage.write_local_file("2024-07-04-raspios-bookworm-arm64-lite.img")
```

I should probably include the script to download/prepare this image.